incubator/keycloak-proxy/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+name: keycloak-proxy
+version: 0.0.1
+appVersion: 3.4.0.Final
```

Comment:
Why not use 3.4.2.Final?

Reply:
@unguiculus That's my mistake. I fixed it and tested.